assets/css/layout/_layout.scss

```diff
@@ -1 +1,2 @@
 // Layout styles.
+@import "main";
```

Comment:
Maybe move the content of `_main.scss` into this file and remove `_main.scss` since `_main` is defining the overall layout.